4.2.5/docker-compose.yml

```
@@ -0,0 +1,159 @@
+version: '3.1'
+
+volumes:
+  geonetwork:
+  esdata:
+  pgdata:
+  pglog:
+  nginxlog:
+  apachelog:
+
+
+networks:
+  gn-network:
+    driver: bridge
+
+
+services:
+  www:
+    image: nginx
+    ports:
+      - 80:80
+    environment:
+      # Only used for / redirect to default webapp
+      - NGINX_PUBLIC_HOST=195.201.225.214
+      #- NGINX_PUBLIC_HOST=apps.titellus.net
+      - NGINX_HOST=195.201.225.214
+      - NGINX_PORT=80
+    volumes:
+      - ./nginx:/etc/nginx/templates
+      - nginxlog:/var/log/nginx
+    depends_on:
+      - geonetwork
+    networks:
+      - gn-network
+
+#  www:
+#    image: httpd
+#    ports:
+#      - 81:80
+#    volumes:
+#      - ./apache/httpd.conf:/usr/local/apache2/conf/httpd.conf
+#      - ./apache/gn-httpd.conf:/usr/local/apache2/conf/gn-httpd.conf
+#      - apachelog:/usr/local/apache2/logs
+#    depends_on:
+#      - geonetwork
+#    networks:
+#      - gn-network
+
+
+  database:
+    image: postgis/postgis:11-2.5
+    environment:
+      POSTGRES_USER: geonetwork
+      POSTGRES_PASSWORD: geonetwork
+      POSTGRES_DB: geonetwork
+    command: [ "postgres",
+               "-c", "log_statement=all",
+               "-c", "logging_collector=true",
+               "-c", "log_file_mode=0644",
+               "-c", "log_directory=/var/log/postgresql",
+               "-c", "log_filename=postgresql.log" ]
+    volumes:
+      - pgdata:/var/lib/postgresql/data
+      - pglog:/var/log/postgresql
+    networks:
+      - gn-network
+
+
+  geonetwork:
+    image: geonetwork:4.2.2
+    restart: always
+    ports:
+      - 8080:8080
+    environment:
+      DATA_DIR: /catalogue-data
+
+      JAVA_OPTS: >
+        -Dorg.eclipse.jetty.annotations.AnnotationParser.LEVEL=OFF
+        -Djava.security.egd=file:/dev/./urandom -Djava.awt.headless=true
+        -Xms512M -Xss512M -Xmx2G -XX:+UseConcMarkSweepGC
+        -Dgeonetwork.resources.dir=/catalogue-data/resources
+        -Dgeonetwork.data.dir=/catalogue-data
+        -Dgeonetwork.codeList.dir=/var/lib/jetty/webapps/geonetwork/WEB-INF/data/config/codelist
+        -Dgeonetwork.schema.dir=/var/lib/jetty/webapps/geonetwork/WEB-INF/data/config/schema_plugins
```

We moved to `/opt/geonetwork` no ?

```suggestion
        -Dgeonetwork.dir=/catalogue-data
        -Dgeonetwork.formatter.dir=/catalogue-data/data/formatter
        -Dgeonetwork.codeList.dir=/opt/geonetwork/WEB-INF/data/config/schema_plugins
        -Dgeonetwork.indexConfig.dir=/opt/geonetwork/WEB-INF/data/config/index
```